lib/matplotlib/animation.py

```diff
@@ -314,13 +317,11 @@ def _adjust_frame_size(self):
             w, h = adjusted_figsize(wo, ho, self.dpi, 2)
             if not (wo, ho) == (w, h):
                 self.fig.set_size_inches(w, h, forward=True)
-                verbose.report('figure size (inches) has been adjusted '
-                               'from %s x %s to %s x %s' % (wo, ho, w, h),
-                               level='helpful')
+                _log.info('figure size (inches) has been adjusted '
+                               'from %s x %s to %s x %s' % (wo, ho, w, h))
```

Comment:
At some point these should be converted to use the multiple arguments syntax of logging rather than the original string substitution syntax.  Do you want to defer that?

Reply:
There's also the issue of reindenting everything.
I think this can wait, it's easy enough to grep for `\b_log.` and fix this in a piecemeal fashion.

Reply:
I just avoided doing this because my formatting skills are poor.  In this case can I just do:
```python
_log.info('figure size (inches) has been adjusted '
                               'from %s x %s to %s x %s', (wo, ho, w, h))
```

Reply:
I'll note its not slowing down relative to before where these formatters were all being executed, so at least by not fixing these I'm not making things worse.

Reply:
the correct fix would be
```
_log.info('figure size (inches) has been adjusted '
                               'from %s x %s to %s x %s', wo, ho, w, h)
```
(no inner parentheses)